```
==================== Start Test: sphere_ffd_def_bspline ====================
/github/workspace/tests/TestCases/deformation/spherical_ffd
sphere_ffd_def_bspline: PASSED
execution command: SU2_DEF def_spherical_bspline.cfg > def_spherical_bspline.log 2>&1
test_iter=10
test_vals (stored): 0.002068
sim_vals (computed): 0.002068
delta_vals: 0.000000
test duration: 0.03 min
Traceback (most recent call last):
  File "/github/workspace/install/bin/continuous_adjoint.py", line 32, in <module>
    import SU2
  File "/github/workspace/install/bin/SU2/__init__.py", line 19, in <module>
    from SU2 import eval
  File "/github/workspace/install/bin/SU2/eval/__init__.py", line 1, in <module>
    from SU2.eval.functions import function as func
  File "/github/workspace/install/bin/SU2/eval/functions.py", line 92
    header_to_key = {v["HEADER"]: k for k, v in su2io.historyOutFields.items()}
    ^
IndentationError: unexpected indent
```

--- a/SU2_CFD/src/output/CFlowOutput.cpp
+++ b/SU2_CFD/src/output/CFlowOutput.cpp
@@ -1789,6 +1789,9 @@ void CFlowOutput::AddAerodynamicCoefficients(const CConfig* config) {
   AddHistoryOutput("AOA", "AoA", ScreenOutputFormat::FIXED, "AOA", "Angle of attack");
 
   AddHistoryOutput("COMBO", "ComboObj", ScreenOutputFormat::SCIENTIFIC, "COMBO", "Combined obj. function value.", HistoryFieldType::COEFFICIENT);
+  // CUSTOM_OBJFUNC is added here so historyMap.py knows how to get its
+  // value, the actual output is COMBO.
+  if (false) AddHistoryOutput("CUSTOM_OBJFUNC", "ComboObj", ScreenOutputFormat::SCIENTIFIC, "COMBO", "Custom obj. function value.", HistoryFieldType::COEFFICIENT);
 }
 
 void CFlowOutput::SetAerodynamicCoefficients(const CConfig* config, const CSolver* flow_solver){

--- a/SU2_PY/SU2/io/historyMap.py
+++ b/SU2_PY/SU2/io/historyMap.py
@@ -313,7 +313,7 @@ history_header_map = {
     "CUSTOM_OBJFUNC": {
         "DESCRIPTION": "Custom obj. function value.",
         "GROUP": "COMBO",
-        "HEADER": "ComboObj",
+        "HEADER": "CustomObj",
         "TYPE": "COEFFICIENT",
     },
     "DEFORM_ITER": {

--- a/SU2_PY/SU2/io/historyMap.py
+++ b/SU2_PY/SU2/io/historyMap.py
@@ -310,6 +310,12 @@ history_header_map = {
         "HEADER": "ComboObj",
         "TYPE": "COEFFICIENT",
     },
+    "CUSTOM_OBJFUNC": {
+        "DESCRIPTION": "Custom obj. function value.",
+        "GROUP": "COMBO",
+        "HEADER": "ComboObj",
+        "TYPE": "COEFFICIENT",
+    },
     "DEFORM_ITER": {
         "DESCRIPTION": "Linear solver iterations for the mesh " "deformation",
         "GROUP": "DEFORM",

